src/components/05_pages/Content/Content.js

```diff
@@ -147,6 +147,7 @@ class Content extends Component {
       action => action.attributes.id === this.state.action,
     )[0];
     this.props.actionExecute(matchingAction, Object.keys(this.state.checked));
+    this.setState({ checked: {}, action: false });
```

Comment:
`action` defaults to `null` - should we set it back to that or is `false` OK?

Reply:
👍 Good point.